srcpkgs/nodeenv/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'nodeenv'
+pkgname=nodeenv
+version=1.6.0
+revision=1
+wrksrc="nodeenv-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="make python3"
+checkdepends="nodejs python3-coverage python3-mock python3-pytest which"
+short_desc="Node.js virtual environment builder"
+maintainer="Joseph Benden <joe@benden.us>"
+license="BSD-3-Clause"
+homepage="https://github.com/ekalinin/nodeenv"
+distfiles="https://github.com/ekalinin/nodeenv/archive/${version}.tar.gz"
+checksum=87b04b0832d38bd825e004e7c8ed82d75524efaa6b8a72e3a55088d93e4e17f7
+make_check=ci-skip
```

Comment:
It's not about CI, locally tests fail in same way, something's wrong with them.

Reply:
Yes. I used `ci-skip` to make the CI not run these tests, but allow anyone without amd64 chip to see that project has failing tests. It will probably still work, but ymmv.

Should I put something different here?

Reply:
`ci-skip` means the tests run fine locally, but are broken in CI, that's not the case here. Ideally fix the tests or mark them as broken with `no` and add a comment why they fail.

Reply:
I've placed some comments in the code. Thanks!!!

Reply:
But it caught real error - on musl `nodeenv --prebuild` fetches wrong binaries.
On musl, this is because it expect `x86_64-pc-linux-musl` where Void provides `x86_64-unknown-linux-musl`, on i686 because it does not hardcode i686 mirror excepion, unlike for musl.

@jbenden, could you complaint upstream about those?

Oh, and nodeenv is missing dependencies: `ModuleNotFoundError: No module named 'pkg_resources'`

Reply:
I've fixed up i686 glibc. I also adjusted musl, so it might just work! I also fixed the module not found error.

Locally this is all working, but it is failing in CI. Not sure yet why...

Reply:
I am not sure why i686 GNU is failing... It fully works in a local i686 VBox VM. It was a tad old on packages, so I upgraded and re-tested - still locally works.

What is different about that i686 CI instance, compared to a VBox 32-bit VM?

Reply:
I think the thing uses networking while running tests... Is that allowed in the CI?